error-codes.md

```diff
@@ -484,6 +514,42 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
     Check the state/monitor/log of the TiFlash server.
 
+### MySQL native error messages
+
+* ERROR 2013 (HY000): Lost connection to MySQL server during query
+
+    The complete error message: `ERROR 2013 (HY000): Lost connection to MySQL server during query`
+
+    You can handle the error in the following ways:
+
+    - Check whether panic is in the log.
+    - Check whether OOM exists in dmesg using `dmesg -T | grep -i oom`.
+    - A long time of no access might also lead to this error. It is usually caused by TCP timeout. If TCP is not used for a long time, the operating system kills it.
+
+* ERROR 1105 (HY000): other error: unknown error Wire Error(InvalidEnumValue(4004))
+
+    The complete error message: `ERROR 1105 (HY000): other error: unknown error Wire Error(InvalidEnumValue(4004))`
+
+    This error usually occurs when the version of TiDB does not match with the version of TiKV. To avoid version mismatch, upgrade all components when you upgrade the version.
```

Comment:
```suggestion
    This error usually occurs when the version of TiDB does not match with that of TiKV. To avoid version mismatch, upgrade all components when you upgrade the version.
```